ci/.travis.yml.multiple_pkgs.example

```diff
@@ -11,7 +11,7 @@ before_install:
   - wget https://raw.githubusercontent.com/tue-robotics/tue-env/master/ci/packages.sh
   # Determine packages modified in this commit
   - ALL=$([ $TRAVIS_EVENT_TYPE == "cron" ] && echo "true")
-  - source packages.sh --pullrequest=$TRAVIS_PULL_REQUEST --branch=$TRAVIS_BRANCH --all=$ALL # add all folders which need to be excluded as an argument
+  - source packages.sh --commit-range=$TRAVIS_COMMIT_RANGE --all=$ALL # add all folders which need to be excluded as an argument
```

Comment:
Is `pullrequest` arg not required for travis?

Reply:
Nope, as all the information is now included in the commit range. Or could you subscribe a situation in which you think is is required?

Reply:
Is there a way to make this better so that for both azure and travis the syntax is the same? Because azure is requiring pullrequest arg and travis not.

Reply:
The pullrequest argument for azure should only be temporally, As the commits assigned to a build is inconsistent. Recently, it looks to be consistent, but more time is needed to be sure.

Reply:
Small remark, the azure rest API only allows for max 200 commits, so this should be taken into account.

Reply:
Alright. This PR will require a part 2 sometime soon.